modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -98,6 +108,19 @@ def upload_documents
 
       private
 
+      def generate_multi_image_pdf(image_list)
+        Prawn::Document.generate(TMP_PDF_PATH) do |pdf|
+          image_list.each do |img|
+            File.open(TMP_IMG_PATH, 'wb') { |f| f.write Base64.decode64(img) }
+            pdf.image TMP_IMG_PATH, fit: [pdf.bounds.right, pdf.bounds.top]
```

Comment:
does `fit` only scale if it doesn't? does it do so proportionally?

Reply:
Unsure about the first question, I'll test that. As for the second question, from the Prawn docs, `:fit	
scale the dimensions of the image proportionally to fit inside [width,height]`

Reply:
Fit does scale up images that are smaller than the pdf bounds along with scaling down images that are larger.